app/src/main/res/values/strings.xml

```diff
@@ -83,15 +83,15 @@
 
     <string name="off_type">Off type</string>
     <string name="annual_leave">Annual leave (14.5 days)</string>
-    <string name="leave_for_marriage">Leave for marriage (3.0 days)</string>
-    <string name="leave_for_child_marriage">Leave for child\'s marriage (1.0 days)</string>
-    <string name="funeral_leave">Funeral leave (3.0 days)</string>
-    <string name="leave_for_care_of_sick_child">Leave for care of sick child (20.0 day)</string>
-    <string name="pregnancy_examination_leave">Pregnancy examination leave (5.0 day)</string>
-    <string name="sick_leave">Sick leave (60.0 days)</string>
-    <string name="miscarriage_leave">Miscarriage leave (50.0 days)</string>
-    <string name="maternity_leave">Maternity leave (180.0 days)</string>
-    <string name="wife_labor_leave">Wife’s labor leave (14.0 days)</string>
+    <string name="leave_for_marriage">Leave for marriage (%s days)</string>
+    <string name="leave_for_child_marriage">Leave for child\'s marriage (%s days)</string>
+    <string name="funeral_leave">Funeral leave</string>
+    <string name="leave_for_care_of_sick_child">Leave for care of sick child (%s days)</string>
+    <string name="pregnancy_examination_leave">Pregnancy examination leave (%s days)</string>
+    <string name="sick_leave">Sick leave (%s days)</string>
+    <string name="miscarriage_leave">Miscarriage leave (%s days)</string>
+    <string name="maternity_leave">Maternity leave (%s days)</string>
```

Comment:
FATAL: [MissingTranslation] "`maternity_leave`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)